csharp/ql/lib/semmle/code/csharp/dataflow/internal/ExternalFlow.qll

```diff
@@ -485,3 +504,63 @@ string asPartialNeutralModel(UnboundCallable c) {
         + parameters + ";" //
   )
 }
+
+private predicate interpretSummary(
+  UnboundCallable c, string input, string output, string kind, string provenance
+) {
+  exists(
+    string namespace, string type, boolean subtypes, string name, string signature, string ext
+  |
+    summaryModel(namespace, type, subtypes, name, signature, ext, input, output, kind, provenance) and
+    c = interpretElement(namespace, type, subtypes, name, signature, ext)
+  )
+}
+
+private class SummarizedCallableAdapter extends SummarizedCallable {
```

Comment:
Maybe a comment that this is where the MaD rows are being used to create SummarizedCallable's (the same for neutrals as well).